### DISCUSSION TITLE
[Feature] feature-name-goes-here

### DISCUSSION THREAD

**Discussion by NiuHK** [immich-app/immich#13827] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

1. I hope to add an option to automatically delete local files when deleting files in the cloud
2. Now in the recycle bin on the mobile phone, batch deletion can only delete files on the server. If you want to delete the mobile phone synchronously, you need to delete them one by one manually. When manually deleting, you also need to confirm whether to delete the files on the server and the mobile phone separately. I hope to add an option to choose to delete local files synchronously when deleting in batches.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Duplicate of #165
